interceptors/auth/auth_test.go

```diff
@@ -9,19 +9,20 @@ import (
 	"testing"
 	"time"
 
+	"github.com/grpc-ecosystem/go-grpc-middleware/v2/metadata"
```

Comment:
This needs to be moved into the 3rd import group with the other imports.

Reply:
Fixed.